<ui version="4.0" >
 <class>FileTransfer</class>
 <widget class="QDialog" name="FileTransfer" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>633</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Remote Destination Settings</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../icons.qrc" >:/icons/icon.svg</iconset>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>&amp;Remote destination:</string>
       </property>
       <property name="buddy" >
        <cstring>m_destination</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_destination" >
       <property name="text" >
        <string>/home/</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_chooseLocationBtn" >
       <property name="text" >
        <string>&amp;Destination...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QGroupBox" name="groupBox_2" >
       <property name="title" >
        <string>&amp;Files and folders to copy:</string>
       </property>
       <layout class="QVBoxLayout" >
        <property name="spacing" >
         <number>0</number>
        </property>
        <property name="leftMargin" >
         <number>9</number>
        </property>
        <property name="topMargin" >
         <number>9</number>
        </property>
        <property name="rightMargin" >
         <number>9</number>
        </property>
        <property name="bottomMargin" >
         <number>9</number>
        </property>
        <item>
         <widget class="QListWidget" name="m_transferList" />
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QGroupBox" name="groupBox" >
         <property name="title" >
          <string>File Permissions</string>
         </property>
         <layout class="QGridLayout" >
          <item row="0" column="0" >
           <widget class="QLabel" name="label_2" >
            <property name="text" >
             <string>Owner:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QCheckBox" name="m_ownerRead" >
            <property name="toolTip" >
             <string>Toggle read permission</string>
            </property>
            <property name="text" >
             <string>R</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="2" >
           <widget class="QCheckBox" name="m_ownerWrite" >
            <property name="toolTip" >
             <string>Toggle write permission</string>
            </property>
            <property name="text" >
             <string>W</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="3" >
           <widget class="QCheckBox" name="m_ownerExecute" >
            <property name="toolTip" >
             <string>Toggle execute permission</string>
            </property>
            <property name="text" >
             <string>X</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="label_3" >
            <property name="text" >
             <string>Group:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QCheckBox" name="m_groupRead" >
            <property name="toolTip" >
             <string>Toggle read permission</string>
            </property>
            <property name="text" >
             <string>R</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="2" >
           <widget class="QCheckBox" name="m_groupWrite" >
            <property name="toolTip" >
             <string>Toggle write permission</string>
            </property>
            <property name="text" >
             <string>W</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3" >
           <widget class="QCheckBox" name="m_groupExecute" >
            <property name="toolTip" >
             <string>Toggle execute permission</string>
            </property>
            <property name="text" >
             <string>X</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" >
           <widget class="QLabel" name="label_4" >
            <property name="text" >
             <string>Others:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" >
           <widget class="QCheckBox" name="m_othersRead" >
            <property name="toolTip" >
             <string>Toggle read permission</string>
            </property>
            <property name="text" >
             <string>R</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2" >
           <widget class="QCheckBox" name="m_othersWrite" >
            <property name="toolTip" >
             <string>Toggle write permission</string>
            </property>
            <property name="text" >
             <string>W</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3" >
           <widget class="QCheckBox" name="m_othersExecute" >
            <property name="toolTip" >
             <string>Toggle execute permission</string>
            </property>
            <property name="text" >
             <string>X</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="4" >
           <widget class="QCheckBox" name="m_makeDirsExecutable" >
            <property name="text" >
             <string>Make directories executable</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3" >
         <property name="title" >
          <string>Misc</string>
         </property>
         <layout class="QVBoxLayout" >
          <item>
           <widget class="QCheckBox" name="m_recursivelyCopyFolders" >
            <property name="text" >
             <string>Recursively copy directories</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="m_btnBox" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons" >
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../icons.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>m_btnBox</sender>
   <signal>rejected()</signal>
   <receiver>FileTransfer</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>611</x>
     <y>276</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_btnBox</sender>
   <signal>accepted()</signal>
   <receiver>FileTransfer</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>543</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
